100of100Service/src/main/java/dev/coms4156/project/HrDatabaseFacade.java

```diff
@@ -18,7 +18,7 @@ public final class HrDatabaseFacade {
   private static DatabaseConnection dbConnection = null;
 
   private final int organizationId;
-  private List<Employee> employees;
+  List<Employee> employees;
```

Comment:
Not sure we need this... Given that this is a final class and cannot be inherited.